src/Component/Frontend/Homepage.tsx

```diff
@@ -2,75 +2,65 @@ import React from 'react';
 import Nav from 'react-bootstrap/Nav';
 import Button from 'react-bootstrap/Button';
 import Card from 'react-bootstrap/Card';
-import Carousel from 'react-bootstrap/Carousel';
-
 export const Homepage = () => {
   return (
-    <>
-      <div className='main-div' style={{ backgroundColor: '#e0e0e0', minHeight: '100vh', borderRadius: '20px', padding: '20px' }}>
-        <div className='navbar' style={{ backgroundColor: 'white', display: 'flex', alignItems: 'center', justifyContent: 'space-between', padding: '10px' }}>
-          <h1 style={{ margin: '0' }}>Welcome To LMS</h1>
-          <Nav variant="underline" defaultActiveKey="/home">
-            <Nav.Item>
-              <Nav.Link href="/home">Active</Nav.Link>
-            </Nav.Item>
-            <Nav.Item>
-              <Nav.Link eventKey="link-1">Option 2</Nav.Link>
-            </Nav.Item>
-            <Nav.Item>
-              <Nav.Link eventKey="disabled" disabled>
-                Disabled
-              </Nav.Link>
-            </Nav.Item>
-          </Nav>
-
 
-     
-        </div>
+    <div className='main-div' style={{ backgroundColor: '#e0e0e0', minHeight: '100vh', borderRadius: '20px', padding: '20px' }}>
```

Comment:
Same comment here as below. improve readability by moving the style properties one to each line